samples/matter/light_bulb/README.rst

```diff
@@ -78,6 +78,13 @@ Device Firmware Upgrade support
     :start-after: matter_door_lock_sample_build_with_dfu_start
     :end-before: matter_door_lock_sample_build_with_dfu_end
 
+FEM support
+===========
+
+.. |fem_file_path| replace:: :file:`samples/matter/common`
```

Comment:
This needs to be removed because of https://github.com/nrfconnect/sdk-nrf/pull/5263. Do not merge before that PR is in.